rdbms/src/main/java/org/spine3/server/storage/jdbc/aggregate/query/InsertEventCountQuery.java

```diff
@@ -31,7 +31,11 @@
 import static org.spine3.server.storage.jdbc.aggregate.query.Constants.EVENT_COUNT_COL;
 import static org.spine3.server.storage.jdbc.aggregate.query.Constants.ID_COL;
 
-
+/**
+ * Query that inserts a new event count to the table with event counts.
```

Comment:
Thanks, cap. Pls be more precise. `Inserts a new aggregate event count after the last snapshot to the corresponding table.`